neuland/digitizing/R3BDigitizingPaddle.h

```diff
@@ -39,8 +39,17 @@ namespace R3B::Digitizing
         double energy{};
         double time{};
         double position{};
-        const Channel::Signal& leftChannel;
-        const Channel::Signal& rightChannel;
+        const Channel::Signal leftChannel;
+        const Channel::Signal rightChannel;
+        Channel::CalSignal leftChannelCal;
+        Channel::CalSignal rightChannelCal;
```

Comment:
When you declare them as values like here, each time you create a paddle signal, it **copies** `Channel::signal` objects, which can be very expensive. Use reference or pointer to eliminate the copy.